cluster/addons/fluentd-elasticsearch/es-image/run.sh

```diff
@@ -22,4 +22,6 @@ export MINIMUM_MASTER_NODES=${MINIMUM_MASTER_NODES:-2}
 
 /elasticsearch_logging_discovery >> /elasticsearch/config/elasticsearch.yml
 
-/elasticsearch/bin/elasticsearch
+chown -R elasticsearch:elasticsearch /data
```

Comment:
Ohhhh, I see my stoopid.  You were su-ing the other way. I'm a fool. Ignore me. I'm going to delete these comments so they don't confuse a future person.